Orange/widgets/utils/itemmodels.py

```diff
@@ -869,8 +869,25 @@ class DomainModel(VariableListModel):
                  ATTRIBUTES)
     PRIMITIVE = (DiscreteVariable, ContinuousVariable)
 
-    def __init__(self, order=SEPARATED, placeholder=None,
+    def __init__(self, order=SEPARATED, separators=True, placeholder=None,
```

Comment:
In general, beware of changing the parameter order or inserting arguments within the existing arguments. If anybody constructed a `DomainModel` by calling `DomainModel(DomainModel.Primitive, "(No color)")` and not `DomainModel(DomainModel.Primitive, placeholder="(No color)")`, this change would break his/her code.

However, we seem to have always given these arguments with keywords.

Reply:
We could also add a *, before them to make it explicit that they can only be used as keyword arguments.

Reply:
I like it. Not just here.

- It makes it easier to manipulate arguments without breaking backward compatibility.
- It forces is to name arguments in calls, which makes the code more readable and prevents making some mistakes we've made in the past.

We should at least encourage adding a `*` in new functions.